app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,217 @@
+  }
+}
+
+h4 {
+  background: rgba(65, 92, 108, 0.5);
+  color: $white;
+  font: bold 24px/45px Helvetica, Sans-Serif;
+  font-variant: small-caps;
+  letter-spacing: 6px;
+  padding: 10px;
+}
+
+ul {
+  list-style-type: none;
+  margin-left: 4px;
+}
+
+.profile-photo {
+  margin-right: 5px;
+  width: 50px;
+}
+
+$facebook-color: #3b5998;
+$twitter-color: #55acee;
+$google-color: #d50f25;
+$text-transform: uppercase;
+$icon-size: 2rem;
+
+.split.button {
+  width: 10%;
+
+  &.facebook {
+    background: $facebook-color;
+    text-transform: $text-transform;
+
+    span {
+      background: darken( $facebook-color, 10% );
+
+      &:after {
+        border: none;
+        font-family: "foundation-icons";
+        content: "\f1c4";
+        font-size: $icon-size;
+        line-height: 0.25rem;
+        margin-left: -0.7rem;
+      }
+    }
+  }
+
+  &.twitter {
+    background: $twitter-color;
+    text-transform: $text-transform;
+
+    span {
+      background: darken( $twitter-color, 10% );
+
+      &:after {
+        border: none;
+        font-family: "foundation-icons";
+        content: "\f1e4";
+        font-size: $icon-size;
+        line-height: 0.25rem;
+        margin-left: -0.7rem;
+      }
+    }
+  }
+
+  &.google {
+    background: $google-color;
+    text-transform: $text-transform;
+
+
+    span {
+      background: darken( $google-color, 10% );
+
+      &:after {
+        border: none;
+        font-family: "foundation-icons";
+        content: "\f1ca";
+        font-size: $icon-size;
+        line-height: 0.25rem;
+        margin-left: -0.7rem;
+      }
+    }
+  }
+
+  &.left-icon {
+    text-align: right;
+    span {
+      left: 0;
+    }
+  }
+}
+.service {
+  width: 100%;
+  height: 320px;
+  margin: 80px 0;
+  text-align: center;
+  border: 1px solid #ddd;
+  -webkit-transition: all 0.3s ease;
+  transition: all 0.3s ease; }
+  .service .service-icon-box {
+    position: relative;
+    top: 100px;
+    display: inline-block;
+    margin-bottom: 40px;
+    padding: 10px;
+    background: white;
+    -webkit-transition: all 0.3s ease;
+    transition: all 0.3s ease; }
+  .service .service-heading {
+    position: relative;
+    top: 80px;
+    -webkit-transition: all 600ms cubic-bezier(0.68, -0.55, 0.265, 1.55);
+    transition: all 600ms cubic-bezier(0.68, -0.55, 0.265, 1.55); }
+  .service .service-description {
+    width: 80%;
+    margin: 0 auto;
+    opacity: 0;
+    -webkit-transition: all 600ms cubic-bezier(0.68, -0.55, 0.265, 1.55);
+    transition: all 600ms cubic-bezier(0.68, -0.55, 0.265, 1.55);
+    -webkit-transform: scale(0);
+    -ms-transform: scale(0);
+    transform: scale(0); }
+  .service .service-icon-box > img.service-icon {
+    width: 40px; }
+  .service:hover {
+    border-color: #00a8ff; }
+  .service:hover .service-icon-box {
+    top: -30px; }
+  .service:hover .service-heading {
+    top: -30px; }
+  .service:hover .service-description {
+    opacity: 1;
+    -webkit-transform: scale(1);
```

Comment:
Properties should be ordered opacity, -ms-transform, -webkit-transform, transform<br>Avoid vendor prefixes.